packages/contracts/contracts/CdpManager.sol

```diff
@@ -416,15 +417,15 @@ contract CdpManager is CdpManagerStorage, ICdpManager, Proxy {
                     totals.totalCollSharesSurplus +
                     singleRedemption.collSurplus;
 
+                bytes32 _nextId = sortedCdps.getPrev(_cId);
```

Comment:
_nextID is always stored, `_cID` is the current ID

If we had a full redemption, we change `_cID` to the next
In the case of partial this will remain current